packages/generator-cmsgov/generators/app/index.js

```diff
@@ -36,14 +36,25 @@ module.exports = class extends Generator {
         message: 'Generate React files too?',
         default: true
       }]);
-    }).then(answers => {
+    })
+    .then(answers => {
       this.props.slug = answers.slug;
       this.props.react = answers.react;
+
+      return this.prompt([{
```

Comment:
I'm not sure how to get to this prompt. It doesn't show it for me.

Reply:
I think you might need to rerun `yarn install` for this new prompt to work

Reply:
![screen shot 2017-09-19 at 3 57 21 pm](https://user-images.githubusercontent.com/7595652/30619264-64f21f96-9d53-11e7-99ff-dfbfdf9dff3a.png)
Heh, I guess we've already started running into problems with node v8.x

Reply:
Following up on this, you have to run `yarn upgrade generator-cmsgov`...which isn't ideal. Not sure if there's a better way around that.